include/wx/graphics.h

```diff
@@ -431,10 +431,13 @@ class WXDLLIMPEXP_CORE wxGraphicsContext : public wxGraphicsObject
 #if wxUSE_PRINTING_ARCHITECTURE
     static wxGraphicsContext * Create( const wxPrinterDC& dc);
 #endif
-#ifdef __WXMSW__
-#if wxUSE_ENH_METAFILE
+#if defined(__WXMSW__) && wxUSE_ENH_METAFILE
```

Comment:
It seemed messy to have two-lines `#ifdef` in the added code and I changed the other one above it for consistency.